contents/ios/IntroToIos.md

```diff
@@ -0,0 +1,93 @@
+<frontmatter>
+  title: Introduction to iOS App Development
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to iOS Development
+
+**Authors: [Bryan Lew](https://github.com/blewjy)**
+**Reviewers: [Chester Sng](https://github.com/ChesterSng), [Yu Pei, Henry](https://github.com/YuPeiHenry), [Jiang Chunhui](https://github.com/Adoby7)**
+
+## What is iOS?
+
+iOS is the mobile operating system that runs on Apple's mobile devices, most notably the iPhone and the iPad. Applications that run on iOS can be downloaded officially from the App Store, and developers of iOS applications can submit their own applications to the App Store to share it with the rest of the world. 
+
+## Why iOS?
+
+In the mobile world, [2 operating systems dominate the whole market share: iOS and Android](https://www.theverge.com/2017/2/16/14634656/android-ios-market-share-blackberry-2016). While Android boasts higher market share in terms of the total number of hardware devices running the operating system, iOS is no slouch in other aspects, especially those important to us sofware engineers.
```

Comment:
How about this (shorter)?

[2 operating systems dominate the mobile device market: iOS and Android](https://www.theverge.com/2017/2/16/14634656/android-ios-market-share-blackberry-2016). While more devices are running Android compared to iOS, given below are some areas in which iOS outshines Android: